vita3k/shader/src/usse_translator_entry.cpp

```diff
@@ -1078,13 +1077,13 @@ void convert_gxp_usse_to_spirv(spv::Builder &b, const SceGxmProgram &program, co
     const uint64_t *secondary_program_start = program.secondary_program_start();
     const uint64_t *secondary_program_end = program.secondary_program_end();
 
-    std::map<ShaderPhase, std::pair<const std::uint64_t *, std::uint64_t>> shader_code;
+    std::array<std::pair<const std::uint64_t *, std::uint64_t>, (size_t)ShaderPhase::Max> shader_code;
```

Comment:
```suggestion
    std::array<std::pair<const std::uint64_t *, std::uint64_t>, (size_t)ShaderPhase::Max> shader_code{};
```
Without this, the content of the array is undefined, which will likely lead to a crash if one of the two phases is not defined.

Reply:
well the whole array is filled in the code below so this shouldn't matter.

Reply:
Indeed, my bad